src/main/js/components/columns/Columns.jsx

```diff
@@ -3,10 +3,29 @@ import AmpersandMixin from 'ampersand-react-mixin';
 import {translate} from 'react-i18next';
 import {getLanguageOfLangtag} from '../../helpers/multiLanguage';
 import {ColumnKinds, FallbackLanguage, LanguageType} from '../../constants/TableauxConstants';
+import * as _ from 'lodash'
+import ColumnEntry from './ColumnEntry.jsx'
+import TableauxConstants from "../../constants/TableauxConstants"
+import Dispatcher from "../../dispatcher/Dispatcher"
+import * as R from 'ramda'
```

Comment:
Didn't know about that one - replaced Ramda usages and removed dependency from package.json. Works great!